src/common/components/map/interactions/HankeClick.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useContext } from 'react';
+import { MapBrowserEvent } from 'ol';
+import OLVectorLayer from 'ol/layer/Vector';
+import { useHistory } from 'react-router-dom';
+import MapContext from '../MapContext';
+import { MapInstance } from '../types';
+
+const HankeClick: React.FC = () => {
+  const { map } = useContext(MapContext);
+
+  const history = useHistory();
+
+  const selectHanke = (mapInstance: MapInstance, evt: MapBrowserEvent<UIEvent>) => {
+    mapInstance?.getLayers().forEach((BaseLayer) => {
+      if (BaseLayer instanceof OLVectorLayer) {
+        BaseLayer.getFeatures(evt.pixel).then((features) => {
+          if (features.length > 0) {
+            features.some((feature) => {
```

Comment:
Mikä tämän somen tarkoitus on?

Reply:
Pilkun viilausta, varmistetaan että featuresia ei enää iteroida kun linkkiä on jo kertaalleen muokattu. Jostain syystä kuvittelin että näin voisi ehkä tapahtua joten en halunnut että query-parametriä puskettaisiin montaa kertaa.